content/test/integration/service/snapshots/SnapshotManager.spec.ts

```diff
@@ -78,7 +80,8 @@ describe('Integration - Snapshot Manager', () => {
     // Assert snapshot was created
     const snapshotMetadata = snapshotManager.getSnapshotMetadata(EntityType.SCENE)
     expect(snapshotMetadata).toBeDefined()
-    expect(snapshotMetadata!.lastIncludedDeploymentTimestamp).toEqual(lastDeploymentTimestamp)
+    expect(lastDeploymentTimestamp).toBeTruthy((a) => !(a instanceof Error))
```

Comment:
Why `Error`? Shouldn't these 2 lines be replaced with `assertResultIsSuccessfulWithTimestamp`?